package.json

```diff
@@ -31,8 +31,10 @@
   },
   "scripts": {
     "clean": "rm -rf test-artifacts",
-    "lint": "eslint .",
-    "test": "yarn clean && jest"
+    "lint": "yarn install-templates && eslint .",
+    "test": "yarn clean && jest",
+    "flow": "yarn install-templates && flow",
+    "install-templates": "yarn install --cwd ./templates/plugin/content"
```

Comment:
Can this happen in a `postinstall` stem instead? Just thinking that it would be better to install these deps inside the docker container, instead of within the run in the pipeline.

Reply:
I'm not sure running `yarn install` in a `postinstall` step will work.  Buildkite run attempts to [install ad infinitum](https://buildkite.com/uberopensource/create-fusion-plugin/builds/115).

Reply:
😱 yeah, that's no good.

What about instead of doing `--cwd`, we `cd templates... && yarn install`? Same problem?

Reply:
Not entirely sure how the directories work inside of the docker package.  I get:

```
$ cd templates/plugin/content && yarn install
--
  | /bin/sh: 1: cd: can't cd to templates/plugin/content
  | error Command failed with exit code 2.

```
or
```
$ cd ./templates/plugin/content && yarn install
--
  | /bin/sh: 1: cd: can't cd to ./templates/plugin/content
  | error Command failed with exit code 2.
```

Reply:
Ah, it seems that's because the directory doesn't exist yet. We would need to switch these two lines: 

https://github.com/fusionjs/create-fusion-plugin/blob/b41815acc9424fae271b12a6e82d9d58a4c127fc/Dockerfile#L8-L10